test/com/facebook/buck/jvm/java/testdata/mixed_javac_plugin_annotation_processors/BUCK.fixture

```diff
@@ -0,0 +1,71 @@
+java_library(
+    name = "main",
+    srcs = [
+        "Main.java",
+    ],
+    plugins = [
+        ":javac_plugin",
+        ":annotation_processor",
+    ],
+)
+
+java_annotation_processor(
+    name = "annotation_processor",
+    processor_class = "com.example.AnnotationProcessor",
+    deps = [
+        ":annotation_processor_lib",
+    ],
+)
+
+java_library(
+    name = "annotation_processor_lib",
+    srcs = [
+        "AnnotationProcessor.java",
+    ],
+    deps = [
+        ":ap_util",
+    ],
+)
+
+java_plugin(
+    name = "javac_plugin",
+    plugin_name = "MyPlugin",
+    deps = [
+        ":javac_plugin_lib",
+    ],
+)
+
+java_library(
+    name = "javac_plugin_lib",
+    srcs = [
+        "JavacPlugin.java",
+    ],
+    deps = [
```

Comment:
@jtorkkola can you please add this requirement to the contributing doc? https://github.com/facebook/buck/blob/master/CONTRIBUTING.md#code-style

This is the first time in reviews this was made a concern. I think its best we set the same standard for everyone to avoid loss of review cycle time

Reply:
😞 Couldn't this be told earlier also? 
Why not setting up a pre-commit hook that will run GJF and Buildifier?

Reply:
I was checking, `BUCK` inside buck src **are not** formatted with Buildifier... (you can check all the changes that would appear if running `find . -type f -name BUCK | xargs buildifier -mode=build`).
So i only will commit the files on the BUCK.fixture ones...

Reply:
We run a Buildifier linter internally with every code review, which also covers `BUCK` files inside buck srcs. Looks like we've either switched this on after the fact, or have been ignoring it lately (I don't have any historical knowledge on this). Will check with the team on whether we want to add this to the contributing doc.

@styurin for the pre-commit hook question.